sessions/stats-analyses-basic.qmd

```diff
@@ -98,30 +103,63 @@ analyses.
 
 <!-- TODO: Create a revealjs presentation on this? -->
 
-:::: callout-note
+:::::::: callout-note
 ## Reading task: \~15 minutes
 
-Almost all fields of research, and definitely more heavily-quantitative
-and scientific fields like biomedicine and health, have math and
-statistics at the core of taking data to draw inferences or general
-observations about the world.
-
-Any time we collect data and need to interpret what it means, we need
-statistics. And anytime we want to make inferences about the world from
-the data, we need to use statistics to determine the likelihood, or
-rather the uncertainty, in those inferences. Statistics is meant to
-quantify uncertainty.
-
-How do we quantify uncertainty? By first creating a "theoretical model"
-that expresses mathematically our research question. For instance, we
-have a theoretical model that outdoor plants grow (non-linearly) with
-water and sunlight, but that more sunlight likely means less water (less
-rain). While *any* research question could be translated into a
-theoretical model, not all theoretical models can be *tested* against
-the real world. And that's where the second part comes in: Our
-theoretical model needs to be structured in a way that allows us to
-measure the items ("parameters") in our model, so that we can build a
-mathematical model from the data ("test it in the real world").
+A statistical model is a simple way to describe the real world using
+mathematics. In research, we often aim to understand the relationships
+between multiple variables. For example, if we change the predictor
+(independent variable) ( x ), how does that affect the outcome
+(dependent variable) ( y )?
+
+Some simple examples are:
+
+-   "How does this new cancer treatment affect survival rates?"
+
+-   "How does this new user interface affect user watch time?"
+
+or more complex:
+
+-   "How does sunlight and water affect plant growth?"
+
+-   "What is the relationship between a metabolite and T1D compared to
+    controls, adjusting for age and gender?"
+
+These relationships can involve single or multiple predictors (such as
+sunlight and water, or metabolite, gender, and age). The outcomes can be
+either continuous (such as plant growth) or categorical (such as whether
+a person has T1D or not).
+
+The relationship between predictors and the outcome is known as a
+regression. If there is a relationship between ( x ) and ( y ), the
+corresponding mathematical object is a function:
+
+::: callout-note
+$$y = f(x) $$
+:::
```

Comment:
```suggestion
$$y = f(x)$$
```

Callout blocks for this situation for equations isn't necessary. We could add an equation number beside though?